steward_proto/proto/steward.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+package steward.v1;
+
+option go_package = "/steward_proto";
+
+import "aave_v2_stablecoin.proto";
+
+service ContractCall {
+    rpc Submit(SubmitRequest) returns (SubmitResponse) {}
+}
+
+message SubmitRequest {
+    string cellar_id = 1;
+    oneof call_data {
+        AaveV2Stablecoin aave_v2_stablecoin = 2;
```

Comment:
If we're moving Uniswap to 100 presumably to get it out of the way to later be deleted, this should probably start at 1.

Reply:
`cellar_id` is 1, or does `oneof` start it's own count?

Reply:
Whoops, totally missed that, no.